# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/sgutils/hotfix-musl-uint.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

sg_dd.c:2402:17: error: unknown type name 'uint'; did you mean 'int'?
 2402 |                 uint off;

--- sg3_utils-1.47/src/sg_dd.c.vanilla	2023-10-11 17:29:27.932587528 +0200
+++ sg3_utils-1.47/src/sg_dd.c	2023-10-11 17:30:02.417588432 +0200
@@ -2399,7 +2399,7 @@
             res = blocks * blk_sz;
             if (iflag.zero && iflag.ff && (blk_sz >= 4)) {
                 uint32_t pos = (uint32_t)skip;
-                uint off;
+                unsigned off;
 
                 for (k = 0, off = 0; k < blocks; ++k, off += blk_sz, ++pos) {
                     for (j = 0; j < (blk_sz - 3); j += 4)
